### ISSUE TITLE
Additional/improved planet generators

### ISSUE THREAD

**Issue by NovaZHart** [NovaZHart/ShooterPrototype#5]
The current planet generator code is very limited and cannot generate a wide variety of planet types. ~~#4 should be done before this, to avoid writing the code twice.~~

- [ ] rings around gas giants
- [ ] planet generator for swirl & line patterns similar to jupiter/saturn
- [ ] continent generator
- [ ] use colormaps
- [ ] lumpy/rocky planets with craters for small moons

**Comment by NovaZHart** [comment]
Removing as duplicate of new issue.
